### PR TITLE
hardening/69_add_user_and_programmer_manual

### PR DESCRIPTION
**Pull request by frbattid**
* Implements issue #69 
* No unit tests nor e2e tests are required
* Asssignee @fgalan

### CODE REVIEW

File: mkdocs.yml [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+extra_css: ["http://fiware.org/style/monokai_sublime.css"]
+pages:
+  - Home: rtfd_home.md
+  - 'User and Programmer manual':
+      - 'Introduction': 'user_and_programmer_manual/introduction.md'
+      - 'Data management and I/O': 'user_and_programmer_manual/batch/data_management_and_io.md'
+      - 'Requesting computing resources': 'user_and_programmer_manual/batch/request_computing_resources.md'
+      - 'Using Hadoop': 'user_and_programmer_manual/batch/using_hadoop.md'
+      - 'OAuth2-protected REST APIs': 'user_and_programmer_manual/batch/oauth2_rest_apis.md'
+  - 'Installation and Administration manual':
```
Comment (fgalan) [inline mkdocs.yml]: I understand this part is yet under work and not part of this PR, right?

Comment (frbattid) [inline mkdocs.yml]: Yes, this will be completed on another PR.

Comment (fgalan) [inline mkdocs.yml]: NTC

**Comment by fgalan** [comment]
LGTM (but only mkdocs.yml and indexes checked, not all the documentation body)
